### PR TITLE
[v6.2] Edit Pass #9

### PR DESCRIPTION
**Pull request by inertial-frame**
This PR picks up where the previous Edit Pass left off.

#6943

---

Infrastructure Section. Moves Terraform page back into TOC. Will likely want to divide up the AWS articles into a new subsection of Guides.

Also the Enterprise Section.

### CODE REVIEW

File: docs/pages/enterprise/workflow/ssh-approval-jira-server.mdx [hunk 1]
```diff
@@ -179,22 +182,22 @@ The `[teleport]` section describes where is the teleport service running, and wh
...
 
 ## Testing
 
 You should be able to run the Teleport plugin now!
 
 ```bash
-teleport-jira start
-INFO   Starting Teleport Access JIRAbot 0.1.0-alpha.3:teleport-jira-v0.1.0-alpha.3-0-gea1ef8e jira/app.go:74
+sudo teleport-jira start
+# INFO   Starting Teleport Access JIRAbot 0.1.0-alpha.3:teleport-jira-v0.1.0-alpha.3-0-gea1ef8e jira/app.go:74
```
Comment (inertial-frame) [inline docs/pages/enterprise/workflow/ssh-approval-jira-server.mdx:200]: FYI this example commented these lines, I followed suit and did the same in the other articles.

File: docs/pages/aws-terraform-guide.mdx [hunk 2]
```diff
@@ -1,56 +1,68 @@
 ---
 title: Teleport High Availability mode on AWS
-description: How to configure Teleport in High Availability mode for AWS deployments.
-h1: Running Teleport Enterprise in High Availability mode on AWS
+description: How to configure Teleport in High Availability mode for AWS deployments
```
Comment (inertial-frame) [inline docs/pages/aws-terraform-guide.mdx]: This period will be reverted.

**Review by klizhentas — APPROVED** [review]

**Comment by inertial-frame** [comment]
@benarent - Last one! Thanks for all the great reviews. Think we ended up making a pretty big dent. Still a lot of work left to go. After this, it'll most be refactoring, validation, and new articles. Several are queued up already. 

Headsup - this PR mostly addresses a bunch of Editorial and Article Layout changes for some of the SSO and Plugins docs. These have more substantial changes coming in here: https://github.com/gravitational/teleport/tree/adam/docs/62sso

File: docs/pages/aws-oss-guide.mdx [hunk 3]
```diff
@@ -565,9 +560,9 @@ sudo certbot certonly --standalone \
                       --email=$EMAIL
 ```
 
-#### Setup Github Auth
+#### Setup Github auth
```
Comment (benarent) [inline docs/pages/aws-oss-guide.mdx:563]: Random Thought, What if we renamed this 'Setup login with Github', or something that makes it clear what Github auth is used for?

**Review by benarent — APPROVED** [review]

**Pull request merged**